README.md

```diff
@@ -40,18 +40,17 @@ Stable releases are packaged on some distributions:
 Otherwise, you can install from source:
 
 ```shell
-$ cargo install --git https://github.com/greshake/i3status-rust i3status-rs
+$ git clone https://github.com/greshake/i3status-rust
+$ cd i3status-rust
+$ cargo install --path .
 ```
 
 By default, this will install the binary to `~/.cargo/bin/i3status-rs`.
 
-or manually:
+For functional themes and icon sets, you also have to copy files from `files/` to `~/.local/share/i3status-rust/`:
```

Comment:
Hmm, I wonder if we should just say to run `install.sh` to install the default themes/icons, otherwise some people may get confused and put their custom themes in `~/.local/share/i3status-rust/` as well?

Reply:
OK